packages/utils/src/withTimeout.ts

```diff
@@ -8,16 +10,6 @@ export class TimeoutError extends Error {
     }
 }
 
-export class AbortError extends Error {
-    readonly code = 'AbortError'
-    constructor(customErrorContext?: string) {
-        super(customErrorContext === undefined
-            ? `aborted`
-            : `${customErrorContext} aborted`)
-        Error.captureStackTrace(this, AbortError)
-    }
-}
-
 export const withTimeout = <T>(
```

Comment:
Would it be possible to use abortable wait in `withTimeout`? Now those method share very similar code.

Reply:
Fixed in https://github.com/streamr-dev/network-monorepo/pull/917/commits/e0491b7e04048c8efef229ee7fff0c8dcb374efe